.github/CHANGELOG.md

```diff
@@ -61,6 +61,7 @@
 1. [HYD] Randomised per actuator flow restrictions at plane init - @Crocket63 (crocket)
 1. [MCDU] Hide stored elements on A/C Status when there are none - @tracernz (Mike)
 1. [FMGC] Fix ident for CD legs - @tracernz (Mike)
+1. [HYD] Accelerations impacts hydraulic reservoir fluid movement - @Crocket63 (crocket)
```

Comment:
Sorry for not noticing this earlier, but could you move this entry to the bottom?

Reply:
Done